tdrs-frontend/src/components/ComboBox/ComboBox.test.js

```diff
@@ -0,0 +1,344 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { render } from '@testing-library/react'
+
+import ComboBox from './ComboBox'
+
+describe('ComboBox', () => {
+  // it('Select element should have value of selected option', () => {
+  //   const props = {
```

Comment:
Should we be keeping this commented code, or can we remove it?

Reply:
Yes, this can be removed. Thank you.